extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMap.kt

```diff
@@ -123,14 +142,14 @@ class MapboxCarMap {
   /**
    * @param mapboxCarMapObserver the instance used in [registerObserver]
    */
-  fun unregisterObserver(mapboxCarMapObserver: MapboxCarMapObserver) {
+  fun unregisterObserver(mapboxCarMapObserver: MapboxCarMapObserver) = apply {
     carMapSurfaceOwner.unregisterObserver(mapboxCarMapObserver)
   }
 
   /**
    * Optional function to clear all observers registered through [registerObserver]
    */
-  fun clearObservers() {
+  fun clearObservers() = apply {
```

Comment:
do we really need to use apply to return the `MapbboxCarMap` after the execution? Would there be any use case?